app/models/contact.rb

```diff
@@ -12,7 +12,14 @@ class Contact < ApplicationRecord
   has_paper_trail
 
   validates :first_name, presence: true
```

Comment:
I think these would be fairly easy to test in the contact_spec.rb

Reply:
yeah good point, will add tests for those